### PR TITLE
Participants pages + Header + other updates

### PR DESCRIPTION
**Pull request by tipanazar**

### CODE REVIEW

File: cms/src/components/TestFrom.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      },
+      name: 'Max',
+      date: new Date(),
+    },
+  });
+
+  // console.log(watch('role')); TEMP
+
+  const handleSubmitForm = (data) => {
+    console.log(data);
```
Comment (MaksVibe) [inline cms/src/components/TestFrom.tsx]: You have some code issues in the code
Screenshot 2023-08-28 at 10 55 07

File: cms/src/components/TestFrom.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        rules={{ required: 'це поле є обовʼязковим' }}
+        minLength={2}
+        type="date"
+      />
+      <AsyncField
+        control={control}
+        clearErrors={clearErrors}
+        name="async"
+        rules={{ required: 'це поле є обовʼязковим' }}
+        options={promiseOptions}
```
Comment (MaksVibe) [inline cms/src/components/TestFrom.tsx]: Screenshot 2023-08-28 at 10 57 14

File: cms/src/app/page.tsx
Comment (MaksVibe) [inline cms/src/app/page.tsx:1]: Do we need here the changes?

File: cms/src/components/CardsContent/CardsContent.tsx
Comment (MaksVibe) [inline cms/src/components/CardsContent/CardsContent.tsx:1]: Fix code issues pls
Screenshot 2023-08-28 at 11 15 43

File: cms/src/components/Input/Input.styles.ts [hunk 3]
```diff
@@ -60,6 +66,14 @@ export const InputWrapper = styled.div<{ checkIsValid: boolean; begIcon: boolean
...
+    isError &&
+    'outline: 2px solid #dc0c31; \
+    & + label#support-label { \
+      visibility: visible; \
+      color: #dc0c31; \
+    }'}     
+
   &:has(input:focus) {
     outline: 2px solid #939393;
     & + label#support-label {
```
Comment (MaksVibe) [inline cms/src/components/Input/Input.styles.ts:80]: is the code working properly?
Screenshot 2023-08-28 at 11 33 54

File: cms/src/components/Input/Input.tsx
Comment (MaksVibe) [inline cms/src/components/Input/Input.tsx:1]: Fix the following issues
Screenshot 2023-08-28 at 11 34 58

File: cms/src/components/ParticipantsForm/ParticipantsForm.styles.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  }
+
+  & div#buttons-wrapper {
+    & > a {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      gap: 8px;
+      width: 100%;
+      /* margin-bottom: 16px; */
```
Comment (MaksVibe) [inline cms/src/components/ParticipantsForm/ParticipantsForm.styles.ts]: do we need the commented code?

File: cms/src/components/ParticipantsForm/ParticipantsForm.tsx
Comment (MaksVibe) [inline cms/src/components/ParticipantsForm/ParticipantsForm.tsx:1]: Fix the code issues
Screenshot 2023-08-28 at 11 36 31

File: cms/src/components/PopUp/PopUp.tsx
Comment (MaksVibe) [inline cms/src/components/PopUp/PopUp.tsx:1]: Fix the issues
Screenshot 2023-08-28 at 11 37 24

File: cms/src/components/SelectField/SelectField.tsx
Comment (MaksVibe) [inline cms/src/components/SelectField/SelectField.tsx:1]: Fix the issues
Screenshot 2023-08-28 at 11 38 08
Screenshot 2023-08-28 at 11 38 14

File: cms/src/components/TestFrom.tsx
Comment (MaksVibe) [inline cms/src/components/TestFrom.tsx:1]: Fix the issues
Screenshot 2023-08-28 at 11 39 46

Screenshot 2023-08-28 at 11 39 37

File: cms/src/helpers/manageParticipantFormValues.ts
Comment (MaksVibe) [inline cms/src/helpers/manageParticipantFormValues.ts:1]: Fix the issues
Screenshot 2023-08-28 at 11 40 34

File: cms/src/redux/participants/operations.ts
Comment (MaksVibe) [inline cms/src/redux/participants/operations.ts:1]: Fix the types issues
Screenshot 2023-08-28 at 11 42 10

File: cms/src/redux/participants/reducer.ts
Comment (MaksVibe) [inline cms/src/redux/participants/reducer.ts:1]: Fix the issues
Screenshot 2023-08-28 at 11 44 37

File: cms/src/redux/projects/projects.slice.ts
Comment (MaksVibe) [inline cms/src/redux/projects/projects.slice.ts:1]: Fix the issue
Screenshot 2023-08-28 at 11 46 14

**Review by MaksVibe — DISMISSED** [review]

File: cms/src/redux/projects/projects.slice.ts
Comment (tipanazar) [reply cms/src/redux/projects/projects.slice.ts:1]: projects is not mine part

File: cms/src/components/SelectField/SelectField.tsx
Comment (tipanazar) [reply cms/src/components/SelectField/SelectField.tsx:1]: this is not mine

**Review by maxkulbako — APPROVED** [review]

**Review by maxkulbako — APPROVED** [review]

**Pull request merged**